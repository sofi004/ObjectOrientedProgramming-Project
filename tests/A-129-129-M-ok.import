linhas=41
colunas=32
1;1|5
1;2|49
1;3|'abc
1;4|25
1;5|43
2;3|23
2;4|24
2;5|5
