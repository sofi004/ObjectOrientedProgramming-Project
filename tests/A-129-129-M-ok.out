Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|5
1;2|49
1;3|'abc
1;4|25
1;5|43
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|5
4;2|49
4;3|'abc
4;4|25
4;5|43
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Insira o conteúdo da célula: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;1|5
4;2|42
4;3|'abc
4;4|44
4;5|45
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|5
1;2|49
1;3|'abc
1;4|25
1;5|43
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 